docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
I'd recommend to omit this platform-specific file and rely on Python as the unifying language with `python -m sphinx` or similar.